src/fragments/start/getting-started/android/integrate.mdx

```diff
@@ -476,34 +476,34 @@ You may want to change the contents of a record. Below, we'll query for a record
   <Block name="Java">
 
    ```java
-  Amplify.DataStore.query(Todo.class, Where.name.eq("Finish quarterly taxes"),
-      matches -> {
-          if (matches.hasNext()) {
-              Todo todo = matches.next();
-              Todo updatedTodo = original.copyOfBuilder()
-                  .title("File quarterly taxes")
-                  .build();
-              Amplify.DataStore.save(updatedTodo,
-                  updated -> Log.i("Tutorial", "Updated item: " + updatedTodo.getName(),
-                  failure -> Log.e("Tutorial", "Update failed.", failure)
-              );
-          }
-      },
-      failure -> Log.e("Tutorial", "Query failed.", failure)
-  );
+  Amplify.DataStore.query(Todo.class, Where.matches(Todo.NAME.eq("Finish quarterly taxes")),
```

Comment:
nit: The indentation of this code block looks off. I think the original indentation looked okay.

Reply:
@eeatonaws corrected the indentation.

Reply:
updated the delete code snippet.